src/main/scala/org/hammerlab/guacamole/commands/GermlineAssemblyCaller.scala

```diff
@@ -79,14 +79,13 @@ object GermlineAssemblyCaller {
      * @param reads  Set of reads to extract sequence from
      * @param startLocus Start (inclusive) locus on the reference
      * @param endLocus End (exclusive) locus on the reference
-     * @param kmerSize Length of the subsequence
      * @param minOccurrence Mininum number of times a subsequence needs to appear to be included
      * @return List of subsequences overlapping [startLocus, endLocus) that appear at least `minOccurrence` time
      */
     def getConsensusKmer(reads: Seq[MappedRead],
                          startLocus: Int,
                          endLocus: Int,
-                         minOccurrence: Int): Iterable[Array[Byte]] = {
+                         minOccurrence: Int): Iterable[ContigSequence] = {
```

Comment:
Yeah, I think we should keep `ContigSequence` to have only the narrow use as an entire contig indexed by genomic locus. Sequences of bytes that are not entire contigs should be something else. I think keeping them as `Array[Byte]` or `IndexedSeq[Byte]` is the most clear but would not be strongly opposed to introducing a new type (maybe `Bases`?) for them